src/graphforge/datasets/sources/snap.py

```
@@ -0,0 +1,108 @@
+            relationship_types=["CONNECTED_TO"],
+            size_mb=25.0,
+            license="Public Domain",
+            category="social",
+            loader_class="csv",
+        ),
+    ]
+
+    for dataset in datasets:
+        try:
+            register_dataset(dataset)
+        except ValueError as e:  # noqa: PERF203
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PERF203`)

Remove unused `noqa` directive

(RUF100)